Makefile

```diff
@@ -156,7 +157,6 @@ clean:
 verify:
 	hack/verify-gofmt.sh
 	hack/verify-gencode.sh
-	hack/verify-vendor.sh
```

Comment:
why remove this verify

Reply:
This check is mainly to determine whether the vendor in the local directory is the latest version. 

The vendor directory has been removed from the code warehouse, so this check can also be removed

Reply:
oh, good to know